src/diff.jl

```diff
@@ -45,6 +45,7 @@ end
 (D::Differential)(x::Union{Num, Arr}) = wrap(D(unwrap(x)))
 (D::Differential)(x::Complex{Num}) = wrap(ComplexTerm{Real}(D(unwrap(real(x))), D(unwrap(imag(x)))))
 SymbolicUtils.promote_symtype(::Differential, T) = T
+SymbolicUtils.isbinop(f::Differential) = false
```

Comment:
What about all the other operators? This is a breaking change.

Reply:
Just in printing, and the follow up PR fixes the printing.